flipperdevices/flipperzero-firmware
# Fix U2F counter endianness

# What's new

Before, the counter was in little endian, moving all over the place. This changeset appears to fix it.

# Verification 

- Go to https://webauthn.bin.coffee/
- Click "Create Credential"
- Click "Get Assertion"
- Look at "Counter: 00000007 Flags: 1" in the second textarea. It should increment by 1 for every "Get Assertion".

# Checklist (For Reviewer)
